common/app/views/main.scala.html

```diff
@@ -91,6 +91,31 @@
             </script>
         }
 
+        @**********************
+          H A C K D A Y - healine fast testing (mch)
+
+          We update the headline with another one.  
+        ************************@
+        @if(request.path == "/uk") {
+            <script type="text/javascript">
+                var anchorElement = document.querySelectorAll('a.fc-item__link[href^="https://www.theguardian.com/politics/2017/nov/29/brexit-bill-of-50bn-no-more-than-what-uk-owes-eu-says-grayling"]')[0];
```

Comment:
You could use

```js
var anchorElement = document.querySelector('.fc-item__link[href^="https://www.theguardian.com/politics/2017/nov/29/brexit-bill-of-50bn-no-more-than-what-uk-owes-eu-says-grayling"]');
```

instead. Should make it a bit faster. If there is any chance to limit the execution of this JS to a hash in the url or a cookie, if would be much appreciated :)

Reply:
@gustavpursche do you mean `querySelectorAll(...)[0]` is slower than `querySelector` or that prefixing the class name by the element type is slower or both?

I will not have time to demo it today so I will not merge it, because I need other changes happening in `ophan` first, but I may demo it later to some editorial people. Regarding limiting the execution of the `JS`, it can't really be done because we  want half of our audience seeing an headline and another the other headline.